### PR TITLE
doc/cephadm/services: the config section of service specs

### PR DESCRIPTION
**Pull request by rkachach**
Fixes: https://tracker.ceph.com/issues/53997

Signed-off-by: Redouane Kachach <rkachach@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [x] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/cephadm/services/index.rst [hunk 1]
```diff
@@ -86,7 +86,10 @@ Service Specification
 =====================
 
 A *Service Specification* is a data structure that is used to specify the
-deployment of services.  Here is an example of a service specification in YAML:
+deployment of services. In addition to parameters such as `placement`, etc
+the user can set initial values of service configuration parameters.
```
Comment (adk3798) [inline doc/cephadm/services/index.rst:90]: I feel like this comes out a bit vague. We should probably be a bit more clear that this sets ceph config options. Basically running beneath the hood `ceph config set <service-name> <param> <value>` for each param/value pair provided and doing so declaratively (as in, we'll keep checking the setting and change it if it has the wrong value). Additionally, you may want to try and incorporate some stuff from the first attempt at documenting this https://github.com/ceph/ceph/pull/44804

**Review by anthonyeleven — CHANGES_REQUESTED** [review]
Commit and PR titles in conventional form please, something like:

`doc/cephadm/services: Document the config section of service specs`

File: doc/cephadm/services/index.rst (hunk 1 above)
Comment (rkachach) [reply doc/cephadm/services/index.rst]: @adk3798 agree, I reworded the doc and added more information. Please, let me know if more clarification is needed.

**Comment by rkachach** [comment]
> Commit and PR titles in conventional form please, something like:
> 
> `doc/cephadm/services: Document the config section of service specs`

Fixed. BTW: in other doc PRs I saw the prefix `doc/cephadm` being used so not sure what's the correct one to be used. In dev we normally use two levels only i.e `mgr/cephadm`.

**Comment by anthonyeleven** [comment]
Oh and the project likes PR/commit messages to be imperatives, so something like

`doc/cephadm/services: Document the config section of service specs`

**Review by anthonyeleven — APPROVED** [review]

**Review by adk3798 — APPROVED** [review]

**Pull request merged**